deploy/sre-prometheus/ocm-agent/100-ocm-agent-cluster-operator-down.PrometheusRule.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    prometheus: sre-proxy-managed-notification-alerts
+    role: alert-rules 
+  name: sre-co-down-managed-notification-alerts
+  namespace: openshift-monitoring
+spec:
+  groups:
+  - name: sre-co-down-managed-notification-alerts
+    rules:
+    - alert: ClusterOperatorDownNotificationSRE
+      expr: max by (namespace, name, reason) (cluster_operator_up{job="cluster-version-operator",name="monitoring",reason="UpdatingPrometheusUserWorkloadFailed"} == 0)
```

Comment:
Are we sure this status condition always means the instances are overloaded?

Reply:
is there a better service log notification? such that encompass all the possible scenarios?

Reply:
this is the only managed notification i can find: https://github.com/openshift/managed-notifications/blob/master/osd/no_resources_to_run_userworkloadmonitoring.json

Reply:
I personally don't know :/ So I'll leave this open for someone else to hopefully know the answer or possibly green light this.